reana_client/api/client.py

```diff
@@ -209,20 +209,22 @@ def create_workflow_from_json(workflow_json, name, access_token,
         raise e
 
 
-def start_workflow(workflow, access_token, parameters):
+def start_workflow(workflow, access_token, parameters, rerun):
```

Comment:
Let's rather call the option `--restart` to keep the naming harmony with the name of the command `start`.